app/controller.py

```diff
@@ -130,12 +130,14 @@ def writeOrConfirmOverwrite(self):
       self.changeToConfirmOverwrite()
       return
     tb.fileWrite()
+    saveMessage = tb.message #Store the save message so it is not overwritten
```

Comment:
Enter the InteractiveFind prompt with CTRL + F. Then type some string that is not in the document so that the message "Find wrapped around" is shown up. Then try to save and it will simply exit interactive find and all messages disappear, so it is unclear whether save worked or not (it did)